rfc/README.md

```diff
@@ -91,3 +91,4 @@ The list of all RFCs can be found here.
 | 53 | [Use Lock-Free Message Queue Improving Hoodie Writing Efficiency](./rfc-53/rfc-53.md) | `UNDER REVIEW` | 
 | 54 | [New Table APIs and Streamline Hudi Configs](./rfc-54/rfc-54.md) | `UNDER REVIEW` | 
 | 55 | [Improve Hive/Meta sync class design and hierachies](./rfc-55/rfc-55.md) | `UNDER REVIEW` | 
+| 56 | [Federated Storage Layer](./rfc-56/rfc-56.md) | `UNDER REVIEW` | 
```

Comment:
This is out-dated. It is good to just phase-1 commit to capture a RFC number and then work on the RFC.